grantnav/frontend/templates/help.html

```diff
@@ -58,14 +83,101 @@ <h2 id="search_box">{% trans "How does the GrantNav search box work?" %}</h2>
+
+  <h3 id="words_phrases">{% trans "Words and phrases" %}</h3>
+  <ul>
+    <li class="red_underline">{% blocktrans %} A single word: e.g. <a href="/search?text_query=people"><code>people</code></a> {% endblocktrans %}</li>
+    <li class="red_underline">{% blocktrans %} Multiple words: e.g. <a href="/search?text_query=young+people+gardens"><code>young people gardens</code></a> (each word does not necessarily have to be present) {% endblocktrans %}</li>
+    <li class="red_underline">{% blocktrans %} Require each word to be found: e.g. <a href="/search?text_query=young+AND+people"><code>young AND people</code></a> {% endblocktrans %}</li>
+    <li class="red_underline">{% blocktrans %} An exact phrase by enclosing it in quotes: e.g. <a href="/search?text_query=%22young+people%22"><code>"young people"</code></a> {% endblocktrans %}</li>
+    <li class="red_underline">{% blocktrans %} Exclude results that contain a word: e.g. <a href="/search?text_query=youth+NOT+clubs"><code>youth NOT clubs</code></a> {% endblocktrans %}</li>
+  </ul>
+
+  <p>{% blocktrans %}Using the above you can effectively filter the records to a single organisation or an exact grant. Try:{% endblocktrans %}</p>
+  <ul>
+    <li class="red_underline"><a href="/search?text_query=%22The+Dulverton+Trust%22"><code>"The Dulverton Trust"</code></a></li>
+    <li class="red_underline"><a href="/search?text_query=%22360G-wolfson-19750%22"><code>"360G-wolfson-19750"</code></a></li>
+  </ul>
+
+  <h3 id="fields">{% trans "Fields" %}</h3>
+  <ul>
+    <li>{% blocktrans %}Search words and phrases limited to a specific field, for example:{% endblocktrans %}
+      <ul>
+        <li class="red_underline">{% blocktrans %}<a href="/search?text_query=title:gardens"><code>title:gardens</code></a> will search the "title" field for the word "gardens"{% endblocktrans %}</li>
+        <li class="red_underline">{% blocktrans %}<a href="/search?text_query=recipientOrganization.postalCode:NW1"><code>recipientOrganization.postalCode:NW1</code></a> will search the for recipients within the "NW1" postcode district (where the field is populated){% endblocktrans %}</li>
+        <li class="red_underline">{% blocktrans %}<a href="/search?text_query=fundingOrganization.name:%22London Councils%22"><code>fundingOrganization.name:"London Councils"</code></a> will search the "Funding Organisation:Name" field for "London Councils"{% endblocktrans %}</li>
+      </ul>
+    </li>
+    <!-- these search functions not working as expected --> <!--
+    <li>{% blocktrans %}Search for a field with no value (or is missing): e.g. <code>missing:amountAppliedFor</code> will search for grants where the "Amount Applied For" field has no value (or is missing altogether){% endblocktrans %}</li>
+    <li>{% blocktrans %}Search for a field that has any non-null value: e.g. <code>exists:grantProgramme.title</code>{% endblocktrans %} will search for grants where there is a value in the "Grant Programme:Title" field</li>
+   -->
```

Comment:
Can these lines be deleted instead of commented out? We will have them in git if we want to add them back.

Reply:
Removed :)